### ISSUE TITLE
Fix color contrast in Skip Nav

### ISSUE THREAD

**Issue by thekaveman** [cal-itp/benefits#1852]
Note: Skip Nav that is currently on https://test-benefits.calitp.org/ does not have a sufficient color contrast.

When designing links, keep in mind that links can be either their original color, or, it can be the `visited` color once the user has clicked it once already - which is purple `#562b97`. So the contrast has to be sufficient for both. In this case, the purple is sufficient, but the original blue is not.

Here's the full error message from an accessibility test tool. They recommend changing the blue from `#045b86` to a darker `#005681`.

>  • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.2:1. Recommendation:  change text colour to #005681.
>    ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
>    ├── #skip-to-content > div
>    └── <div class="container">Skip to Main Content</div>

cc @indexing @segacy1

_Originally posted by @machikoyasuda in https://github.com/cal-itp/benefits/issues/1652#issuecomment-1686887324_

**Comment by segacy1** [comment]
Hi! I have [design explorations](https://www.figma.com/file/SeSd3LaLd6WkbEYhmtKpO3/Benefits-(Full-Application)?type=design&node-id=13142-884&mode=design&t=8NnXksLQaXVZzb35-4) for skip navs with white, blue, or black backgrounds with high-contrast text and focus rings. I think they can all work. If there is not preference, I'll just pull the blue option (#3) and turn it into a component for development. Please comment or thumbs up! @machikoyasuda @thekaveman @indexing @angela-tran

**Comment by angela-tran** [comment]
I like the lighter blue background too

**Comment by thekaveman** [comment]
@indexing will make the final call this week, and we'll flag for dev implementation.

Thank you @segacy1!

**Comment by indexing** [comment]
Segacy and I reviewed this issue in our design review today. Since we may evolve this `skip nav component` down the road, we are going to keep the scope as small as possible to ensure the current implementation meets accessibility standards.

Here's the plan:

- We are changing the background for the skip nav from yellow to our (existing) secondary background color. 
 
This single change allows us to keep the `active` and `visited` link colors what they are now, meet accessibility standards, and not impact the whole of the application. Secondly, it allows us to add a focus ring to the link. Because the focus ring is also yellow, it wouldn't be visible against the current background. Thirdly, we don't have to expand current styles to support an edge case.

The tradeoff in this approach is that the yellow background functions well to draw the eye of the user upward when they are using keyboard navigation. Given that navigation of the app using this method is more of an edge case, we believe this change will impact only a few users. 

@segacy1 will post a link to the final design in Figma.

**Comment by segacy1** [comment]
Hey all! Linking to the final design in [figma](https://www.figma.com/file/SeSd3LaLd6WkbEYhmtKpO3/Benefits-(Full-Application)?type=design&node-id=8798-49566&mode=design&t=AQcEWBPzF8jGwNc5-4)

**Comment by indexing** [comment]
Unless you have any final thoughts or concerns @machikoyasuda, this issue is ready for implementation.
